cdk/lib/security-hq.ts

```diff
@@ -1,17 +1,170 @@
+import {
+  ComparisonOperator,
+  Metric,
+  TreatMissingData,
+} from '@aws-cdk/aws-cloudwatch';
+import type { CfnTable } from '@aws-cdk/aws-dynamodb';
+import { InstanceClass, InstanceSize, InstanceType } from '@aws-cdk/aws-ec2';
+import type { CfnTopic } from '@aws-cdk/aws-sns';
 import { CfnInclude } from '@aws-cdk/cloudformation-include';
+import { Duration } from '@aws-cdk/core';
 import type { App } from '@aws-cdk/core';
+import { AccessScope, GuApplicationPorts, GuEc2App } from '@guardian/cdk';
+import { GuAlarm } from '@guardian/cdk/lib/constructs/cloudwatch';
 import type { GuStackProps } from '@guardian/cdk/lib/constructs/core';
-import { GuStack, GuStageParameter } from '@guardian/cdk/lib/constructs/core';
+import {
+  GuDistributionBucketParameter,
+  GuStack,
+} from '@guardian/cdk/lib/constructs/core';
+import {
+  GuDynamoDBReadPolicy,
+  GuDynamoDBWritePolicy,
+} from '@guardian/cdk/lib/constructs/iam';
 
+/**
+ * Migration steps:
+ *
+ * - ensure new stack is working okay
+ * - make DNS switch (defined in domains account)
+ * - include full definitions for adopted resources (dynamo, alarm topic) + stop
+ *   importing old template
+ * - finally delete old template + stack
+ */
 export class SecurityHQ extends GuStack {
   migratedFromCloudFormation = true;
 
   constructor(scope: App, id: string, props: GuStackProps) {
     super(scope, id, props);
 
-    new CfnInclude(this, `security-hq-PROD`, {
+    // Import old stuff. Note, this template has been modified slightly to avoid
+    // a clash on the VpcId parameter - as the new service runs in a different
+    // VPC to the old.
+    const template = new CfnInclude(this, `security-hq-PROD`, {
       templateFile: '../cloudformation/security-hq.template.yaml',
-      parameters: { Stage: GuStageParameter.getInstance(this) },
+      parameters: {
+        Stage: this.stage,
+      },
+    });
+
+    // Import the existing template, though note we have removed some
```

Comment:
Minor; has this comment been taken out of context accidentally? 

```suggestion
    // Import the existing Dynamo table
```